### ISSUE TITLE
Which tool was used to create the C arrays from the png images?

### ISSUE THREAD

**Issue by sanamon** [MichaelRColton/PSDR#11]
And is there a better way to communicate with other developers on the same projects, I suppose creating issues isnt the optimal way?  I'm new to github and cannot find any way to send messages to other users?

**Comment by pwarren** [comment]
For my OFF image, I used:
https://www.branah.com/text-to-image
to generate the PNG file. Then as per  PSDR/Source/src/images.c I used http://www.rinkydinkelectronics.com/t_imageconverter565.php to get the C Array. Then I removed '0x00' Bytes from the end until it worked, I'm sure there's a method of calculating the padding correctly, but trial and error did work!

In terms of discussion, Michael's asked that we use github for now. The wiki could be used for some things too.

I did have a Discourse instance set up, but it has seen no use!
